user/src/com/google/gwt/core/CoreWithUserAgent.gwt.xml

```diff
@@ -12,22 +12,13 @@
 <!-- implied. License for the specific language governing permissions and   -->
 <!-- limitations under the License.                                         -->
 
-<!-- Deferred binding rules for core classes based on user agent. -->
+<!-- Deferred binding rules for core classes. -->
```

Comment:
If this is no longer based on user agent, can we move the remaining replace-with to StackTrace.gwt.xml where it probably belongs?

That would leave this file empty, and we can let users remove it from their own projects, eventually removing it here.